src/styles.scss

```diff
@@ -0,0 +1 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
```

Comment:
I think this file is some sort of artifact. Remove it, please. It doesn't look like it is needed anymore.

Reply:
Yeah. It's imported in two files. Removed